lib/hosting/functionsProxy.js

```diff
@@ -67,7 +67,7 @@ module.exports = function(options) {
           Cookie: sessionCookie,
         },
         followRedirect: false,
-        timeout: 60000,
+        timeout: 540000,
```

Comment:
The Hosting functions proxy has a maximum 60s timeout in deployed state, not just in local serving. You should not use Firebase Hosting to proxy to long-running functions.

If you revert this line, I'll accept the patch for the emulator startup.

Reply:
Done!